thirdparty-api/data-models.md

```diff
@@ -0,0 +1,953 @@
+
+A data model representing an [AttestationStatement](https://w3c.github.io/webauthn/#attestation-statement).
+
+| Name | Cardinality | Type | Description |
+| --- | --- | --- | --- |
+| clientDataJSON    | 1 | string | JSON string with client data |
+| attestationObject | 1 | string | CBOR.encoded attestation object|
+
+##### 3.2.1.32 AuthenticatorAssertionResponse
+
+A data model representing an [AuthenticatorAssertionResponse](https://w3c.github.io/webauthn/#authenticatorassertionresponse).
+
+| Name | Cardinality | Type | Description |
+| --- | --- | --- | --- |
+| authenticatorData | 1 | string | Information about the authenticator. |
+| clientDataJSON    | 1 | string | base64 encoded JSON string containing information about the client. |
+| signature         | 1 | string | The signature generated by the private key associated with this credential. |
+| userHandle        | 0..1 | string | This field is optionally provided by the authenticator, and represents the user.id that was supplied during registration.|
+
+
+#### 3.2.2 Enumerations
+##### 3.2.2.1 AuthenticationResponse
+The AuthenticationResponse enumeration describes the result of authenticating a FIDO challenge.
+| Name | Description |
+| ---  | ----------- |
+| VERIFIED | The challenge was correctly signed. |
+| REJECTED | The challenge was not correctly signed. |
+##### 3.2.2.2 AuthorizationResponse
+The AuthorizationResponseType enumeration is the same as the AuthorizationResponse enumeration described in [Section 7.5.3](https://github.com/mojaloop/mojaloop-specification/blob/master/fspiop-api/documents/v1.1-document-set/API%20Definition%20v1.1.md#753-authorizationresponse) of Ref. 1 above.
+##### 3.2.2.3 ConsentRequestChannelType
+
+| Name | Description |
+| ---  | ----------- |
+| WEB | DFSP can support authorization via a web-based login |
+| OTP | DFSP can support authorization via a One Time PIN |
+
+##### 3.2.2.4 ConsentStatusType
+The ConsentStatusType enumeration describes the allowed status values that a consent item can have. These are as follows:
+| Name | Description |
+| ---  | ----------- |
+| PENDING | The consent item has been proposed but not yet approved. |
+| VERIFIED | The consent item has been verified and approved. |
+
+##### 3.2.2.5 CredentialStatus
+This contains the allowed values for the CredentialStatus
+| Name | Description |
+| ---  | ----------- |
+| PENDING | The credential has been created but not yet verified. |
+| VERIFIED | Authentication service has verified the credential. |
+
+##### 3.2.2.6 CredentialType
+The CredentialType enumeration contains the allowed values for the type of credential which is associated with a permission.
+| Name | Description |
+| ---  | ----------- |
+| FIDO | The credential is based on a FIDO challenge. Its payload is a FIDOPublicKeyCredentialAttestation object. |
+| GENERIC | The credential is based on a simple public key validation. Its payload is a GenericCredential object |
+
+##### 3.2.2.7 PartyIdType
```

Comment:
Suggesting to raise a change request on FSP Interoperability API to align the PartyIdType. At least I don't have anything against adding THIRD_PARTY_LINK.